### ISSUE TITLE
Evaluation on public dataset

### ISSUE THREAD

**Issue by Blank-z0** [facebookresearch/generative-recommenders#94]
Hi, great work!
I'm trying to reproduce the results on public datasets. However, I only found the training codes, where the model was evaluated on the eval set (or you don't use train/eval/test spilt, only train/test split?). I’d like to know if you partitioned the public dataset into a test set, and whether the results reported in the paper correspond to the test set or the eval set. 
If I want to partition a test set, should I set `ignore_last_n=0,1,2` when loading the test, eval and train dataset? For example:
```
train_dataset = DatasetV2(
      ratings_file=dp.output_format_csv(),
      padding_length=max_sequence_length + 1,  # target
      ignore_last_n=2,
      chronological=chronological,
)
eval_dataset = DatasetV2(
      ratings_file=dp.output_format_csv(),
      padding_length=max_sequence_length + 1,  # target
      ignore_last_n=1,
      chronological=chronological,
)
test_dataset = DatasetV2(
      ratings_file=dp.output_format_csv(),
      padding_length=max_sequence_length + 1,  # target
      ignore_last_n=0,
      chronological=chronological,
)
```

**Comment by isofun** [comment]
hello i'm reading this code and i think you should set all ignore_last_n to 0. if you store the different datasets in separate file, you need to set different filename for 'ratings_file' as you need. hope it helps.

**Comment by jiaqizhai** [comment]
Thanks for your interest in our work! Please check https://github.com/facebookresearch/generative-recommenders/issues/8
